### PR TITLE
NanoAOD updates to use MiniAODV2 information (same as #22728)

### PR DESCRIPTION
**Pull request by arizzi**
This uses the new content of MiniAODV2 for NanaAOD processing and fixes few things

Needed for NanoAOD production chained with MiniAOD V2

fwd port of #22728

needs: cms-sw/cmsdist#3873
@gpetruc

### CODE REVIEW

**Comment by fabiocos** [comment]
I see minor changes in the electron, muon and jet NanoAOD plots, while the others look unchanged as they should. @arizzi @gpetruc Do you confirm they are according your expectations? The needed external has been merged in the meanwhile by @smuzaffar

**Comment by smuzaffar** [comment]
@fabiocos , external just adds new files which non of the existing code should be using, so merging that should not have any impact.

File: PhysicsTools/NanoAOD/python/jets_cff.py [hunk 1]
```diff
@@ -24,11 +24,21 @@
...
+			  filterParams=cms.PSet(
+			    version = cms.string('WINTER17'),
+			    quality = cms.string('TIGHTLEPVETO'),
+			  ),
+                          src = cms.InputTag("slimmedJets")
+)
+
 looseJetIdAK8 = cms.EDProducer("PatJetIDValueMapProducer",
 			  filterParams=cms.PSet(
 			    version = cms.string('WINTER16'),
```
[the remark this answers is not in the document]
Comment (gpetruc) [reply PhysicsTools/NanoAOD/python/jets_cff.py]: yes, the loose ID exists only in Winter16 (and thus we run it only in the 80X sequence for 2016 data)
similarly, tight lep veto exists only for 2017 and so we don't run it on 2016.

[the remark this answers is not in the document]
Comment (fabiocos) [reply PhysicsTools/NanoAOD/python/jets_cff.py]: ok thanks

**Comment by fabiocos** [comment]
+1

**Pull request merged**